examples/platform_view/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -234,9 +222,12 @@
 			files = (
 			);
 			inputPaths = (
+				"${PODS_ROOT}/Target Support Files/Pods-Runner/Pods-Runner-frameworks.sh",
+				"${PODS_ROOT}/../Flutter/Flutter.framework",
 			);
 			name = "[CP] Embed Pods Frameworks";
 			outputPaths = (
+				"${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/Flutter.framework",
```

Comment:
Why do only some of the projects have these input/output path changes?

Reply:
CocoaPods adds this in their scripting, so it's present in apps that have iOS platform code in their plugins.

Reply:
(so projects that have Podfiles)